src/globalInfo/types.ts

```diff
@@ -44,8 +45,28 @@ export interface SfAliases {
   [alias: string]: string;
 }
 
+export type SfSandbox = {
```

Comment:
This structure is maximized for sandboxes created via sf, retaining all key information. This will save time when operating against the sandbox